CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ReplicatorConfigurationRequestHandler.swift

```diff
@@ -256,3 +284,160 @@ public class ReplicatorConfigurationRequestHandler {
         return true
     }
 }
+
+private class LocalWinCustomConflictResolver: ConflictResolverProtocol {
+    func resolve(conflict: Conflict) -> Document? {
+        let localDoc = conflict.localDocument!
+        let remoteDoc = conflict.remoteDocument!
+        let docId = conflict.documentID
```

Comment:
need to make sure either the local doc or the remote doc is not null

Reply:
same as above

Reply:
Added null check